freedomofpress/securedrop-sdk
# Add support for /users endpoint

## Description

Resolves #131 

Adds support for the `/users` endpoint.

## Status

Ready for review

## Test plan
After visual review:

1. Start an up-to-date development environment 
2. Remove the `test-setup.yml` and `test-get-users.yml` cassettes
3. Re-run tests via `make test`
- [ ] Observe that tests are passing (if testing repeatedly, you may hit the auth bug that will be fixed by #135)
- [ ] Inspect the generated `test-get-users.yml` and observe that it contains the expected result (users on your dev server).

Optionally, run the same test in Qubes via a proxy VM (the cassette name to inspect is `test_get_users.json`).
